mne/event.py

```diff
@@ -746,7 +746,7 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.,
         A raw object to use the data from.
     id : int
         The id to use.
-    start : float
+    start : int | float
```

Comment:
Isn't there a metaclass? @Eric89GLX I remember you told me what to do in such a case before.